tests/2016-09-01-preview/Microsoft.Solutions.tests.json

```diff
@@ -0,0 +1,52 @@
+{
+  "tests": [
+    {
+      "name": "appliances - basic",
+      "definition": "http://schema.management.azure.com/schemas/2016-09-01-preview/Microsoft.Solutions.json#/resourceDefinitions/appliances",
+      "json": {
+        "type": "Microsoft.Solutions/appliances",
+        "name": "[parameters('dnsLabelPrefix')]",
+        "apiVersion": "2016-09-01-preview",
+        "kind": "ServiceCatalog",
+        "location": "[resourceGroup().location]",
+        "properties": {
+          "applianceDefinitionId": "/subscriptions/d05f4e58-b80d-4ebd-a6d7-c9cac216cd39/resourceGroups/EuapAppDefs/providers/Microsoft.Solutions/applianceDefinitions/SinlgeVMBuildDemo",
+          "managedResourceGroupId": "[concat('/subscriptions/', subscription().subscriptionId, '/resourceGroups/', resourceGroup().name, parameters('dnsLabelPrefix'), '-', uniqueString(concat(resourceGroup().name, parameters('dnsLabelPrefix'))))]",
+          "parameters": {
+            "adminUsername": {
+              "value": "[parameters('adminUsername')]"
+            },
+            "adminPassword": {
+              "value": "[parameters('adminPassword')]"
+            },
+            "dnsLabelPrefix": {
+              "value": "[parameters('dnsLabelPrefix')]"
+            }
+          }
+        }
+      }
+    },
+    {
+      "name": "applianceDefinitions - basic",
+      "definition": "http://schema.management.azure.com/schemas/2016-09-01-preview/Microsoft.Solutions.json#/resourceDefinitions/applianceDefinitions",
+      "json": {
+        "type": "Microsoft.Solutions/applianceDefinitions",
+        "name": "[parameters('dnsLabelPrefix')]",
+        "apiVersion": "2016-09-01-preview",
+        "location": "eastus2euap",
+        "properties": {
+          "lockLevel": "None",
```

Comment:
Check Microsoft.Solutions.json, "None" is an allowed value. If we don't allow it, we should update swagger and the schema.